plugins/BEdita/Core/tests/TestCase/Model/Entity/StreamTest.php

```diff
@@ -51,19 +51,19 @@ class StreamTest extends TestCase
     ];
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
      */
-    public function setUp(): void
+    public function setUp()
```

Comment:
This change MUST be reverted in order to work with the latest PHUnit version:

```suggestion
    public function setUp(): void
```